packages/common-sdk/src/web3/network/account-requests.ts

```diff
@@ -1,7 +1,6 @@
-import { Address } from "@project-serum/anchor";
 import { AccountInfo, Connection, PublicKey } from "@solana/web3.js";
 import invariant from "tiny-invariant";
-import { AddressUtil } from "../address-util";
+import { Address, AddressUtil } from "../address-util";
```

Comment:
good catch!

then we need to publish common-sdk with this bug fix ?

Reply:
Yeah, best to publish a new version of the common-sdk. I also think we should consider getting rid of yarn workspaces for this repository and just have each package manage dependencies independently to avoid issues like this in the future.